packages/aragon-wrapper/src/index.js

```diff
@@ -331,6 +340,15 @@ export default class Aragon {
     // Get all app proxy addresses
     const baseApps$ = this.permissions.pipe(
       map(Object.keys),
+      // Dedupe until apps change
+      distinctUntilChanged((oldProxies, newProxies) => {
+        if (oldProxies.length !== newProxies.length) {
+          return true
```

Comment:
Shouldn't this be false? The comparison function returns a bool of whether two items are the same (based on the example -https://rxjs-dev.firebaseapp.com/api/operators/distinctUntilChanged)

Reply:
It's badly documented, but based on the example the comparison functions is `isEqual` rather than `isDistinct`

Reply:
Good catch! Fixed in e6d3afc.